Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfMinFilterMode.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
+{
+    /// <summary>
+    /// Minification filter mode.
+    /// https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/sampler.schema.json
+    /// </summary>
+    public enum GltfMinFilterMode
+    {
+        None = 0,
```

Comment:
Please ass summary tags.

Reply:
Summary already contains enum definition location. It will be more up to date than comments here.